ramls/dto/result.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Result DTO Schema",
```

Comment:
better to rename the file caption from "result.json" to "resultDto.json", even if it's in "dto" directory